yt_dlp/extractor/wimbledon.py

```diff
@@ -0,0 +1,60 @@
+from .common import InfoExtractor
+from ..utils import (
+    parse_duration,
+    traverse_obj,
+)
+
+
+class WimbledonIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?wimbledon\.com/en_GB/video/media/(?P<id>[0-9]+).html'
```

Comment:
Allow other languages/locales; seems many other work in browser.
You could also add an `'only_matching': True` test for an `en_US` URL etc
```suggestion
    _VALID_URL = r'https?://(?:www\.)?wimbledon\.com/\w+/video/media/(?P<id>\d+)\.html'
```

Reply:
Done, added `'only_matching': True` tests for `en_US` and `es_Es`.